Source/UZKFileInfo.m

```diff
@@ -65,6 +97,24 @@ - (NSDate *)timestamp {
 
 
 
+#pragma mark - Private Class Methods
+
+
++ (BOOL)itemIsDirectory:(unz_file_info64 *)fileInfo {
+    UZKZipOS zipOSClassMapping = fileInfo->version >> 8;
+    if (zipOSClassMapping == UZKZipOSMSDOS || zipOSClassMapping == UZKZipOSWindowsNT) {
+        return 0x01 == (fileInfo->external_fa >> 4) && ![UZKFileInfo itemIsSymbolicLink:fileInfo];
```

Comment:
Let's add another wrapper for the `fileInfo->external_fa >> 4` field extraction - what's the name of that field in the spec? Also, what's the significance of the `0x01` value? We should have another enum like we did for UZKZipOS to give names to the possible values and make this more readable.

Also, when I was writing the test cases, I couldn't find a way (using the `zip` command) to get a symlink to a directory in a zip file. Are you able to do that? If you can't figure out how to either, let's remove the itemIsSymbolicLink checks here and on line 109.